epoch-verifier/src/data/context.ts

```diff
@@ -40,4 +47,16 @@ export function individualDonationsByProposals(context: Context): Map<Address, b
 
 export function rewardsByProject(context: Context): Map<Address, Reward> {
   return new Map(context.rewards.map((r) => [r.proposal, r] as const))
-}
\ No newline at end of file
+}
+
+function transformToAllocations(allocationRecords: AllocationRecord[]): Allocation[] {
+  const allocations: Map<Address, Allocation> = new Map()
+
+  for (const record of allocationRecords) {
+    const prev = allocations.get(record.user) ?? { user: record.user, donations: [] }
+    prev.donations.push({ amount: record.amount, proposal: record.proposal })
```

Comment:
BE endpoint uses `allocations` instead of `donations`. I suggest unification to `allocations`.
BE endpoint uses `address` instead of `proposal`. I suggest unification to `address`.

```suggestion
    prev.allocations.push({ amount: record.amount, address: record.proposal })
```

Reply:
Turns out this is not true, because `GET /allocations/epoch/{epoch}` uses different naming than `GET /allocations/user/{user_address}/epoch/{epoch}`.